op/model.py

```diff
@@ -237,6 +294,63 @@ def __init__(self, backend):
     def _load(self):
         return self._backend.config_get()
 
+class Status(ABC):
```

Comment:
How do we benefit from this being an ABC in practice?

Reply:
It allows us to use `@abstractmethod` but we have no use for it here yet.

I will remove inheritance from ABC then.

Reply:
Still there.. I suggest not marking as resolved before the issue is actually resolved, so it's easier to keep track of things.